easybuild/easyblocks/n/numpy.py

```diff
@@ -79,7 +79,10 @@ def configure_step(self):
             "search_static_first=True",
         ])
 
-        if get_software_root("imkl"):
+        # If FlexiBLAS has a dependency on MKL we want to link to FlexiBLAS and not to MKL.
+        imkl_direct = get_software_root("imkl") and not get_software_root("FlexiBLAS")
```

Comment:
The comment and the code doesn't really match up.
The code does not check if FlexiBLAS has a dependency on MKL.
All it does is check if FlexiBLAS is available and if so ignores MKL.

But that still ens up doing what it should, so the comment should be changed to something more relevant.

Reply:
Do you know if there is a way to check the dependencies of an EasyConfig?

Right, in the comment I described when/why this check is needed, but not what the code is doing. Let me try to improve the comment.

Reply:
@akesandgren Do you think this improves the comment?

```suggestion
        # If both FlexiBLAS and MKL are found, we assume that FlexiBLAS has a dependency on MKL.
        # In this case we want to link to FlexiBLAS and not directly to MKL.
        imkl_direct = get_software_root("imkl") and not get_software_root("FlexiBLAS")
```

Reply:
At least it is more in line with what the code does.